src/core/Cargo.toml

```diff
@@ -78,19 +78,23 @@ harness = false
 
 ## Wasm section. Crates only used for WASM, as well as specific configurations
 
-[target.'cfg(all(target_arch = "wasm32", target_vendor="unknown"))'.dependencies.wasm-bindgen]
+[target.'cfg(all(target_arch = "wasm32", target_os="unknown"))'.dependencies.wasm-bindgen]
```

Comment:
- the pure wasm target is `wasm32-unknown-unknown` (used for the NPM package)
- the pyodide wasm target is `wasm32-unknown-emscripten`
- wasi is target `wasm32-wasi`

so setting `os` instead of `vendor` here matches `wasm32-*-unknown` instead of `wasm32-unknown-*` and avoid restricting `POSIX/libc`-like targets like `emscripten` and `wasi`